bedevere/bpo.py

```diff
@@ -4,7 +4,16 @@
 
 
 router = routing.Router()
-
+re1='(<!--.*?-->)'	# HTML Comment 1
+re2='.*?'	# Non-greedy match on filler
+re3='((?:http|https)(?::\\/{2}[\\w]+)(?:[\\/|\\.]?)(?:[^\\s"]*))'	# HTTP URL 1
+re4='.*?'	# Non-greedy match on filler
+re5='(<!--.*?-->)'	# HTML Comment 2
+BPO_URL_RE = re.compile(re1+re2+re3+re4+re5,re.IGNORECASE|re.DOTALL)
+BPO_MSG = """{0}
+<!-- issue number -->
+{1}
```

Comment:
Would it make sense to have a fancier message? E.g. "(Please only discuss the code in this pull request here on GitHub; all other discussion should occur on the issue at [bpo-{issue_number}](https://bugs.python.org/issue{issue_number}).)"